sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AcsChatMessageEventBaseProperties.java

```diff
@@ -45,7 +45,7 @@
      * The version of the message
      */
     @JsonProperty(value = "version")
-    private Integer version;
+    private Long version;
```

Comment:
We shouldn't manually be updating a generated file?

Reply:
This change is because the swagger is changed and code is re-generated. Not a manual change.